containers/azure-terraform/.devcontainer/devcontainer.json

```diff
@@ -1,7 +1,16 @@
 {
-	"name": "Azure Terraform 0.12",
-	"dockerFile": "Dockerfile",
-	"mounts": [ "source=/var/run/docker.sock,target=/var/run/docker.sock,type=bind" ],
+	"name": "Azure Terraform",
+	"build": {
+		"dockerfile": "Dockerfile",
+		"args": { 
+			"TERRAFORM_VERSION": "0.12.16",
```

Comment:
We should update to the latest version **0.12.29**

Reply:
Sounds good.  Also bumped up tflint to latest (`0.18.0`).